pkg/client/client.go

```diff
@@ -279,6 +348,43 @@ func checkAwaitingSignature(
 	}
 
 	if isAwaitingDigest {
+		if err := requestedSignatures.add(keepAddress, latestDigest); err != nil {
+			logger.Errorf("failed to register signature request [%v]", err)
```

Comment:
If I'm not wrong we have a convention to add a `:` before we print `err`. So I think it should be: `logger.Errorf("failed to register signature request: [%v]", err)`

Reply:
Agree with the comment above but this line needs to be refactored anyway after #372 is merged and we'll be no longer handling error here.